docs/backend/notification.md

```diff
@@ -2,24 +2,24 @@
 title: Notifications
 ---
 
- # Notifications
+# Notifications
```

Comment:
These markdown prettifications should disappear from this PR once I merge upstream master into my branch as it's been taken care of in #4599.